interfaces/devdoc/timer_requirements.md

```diff
@@ -0,0 +1,96 @@
+# timer
+================
+
+## Overview
+
+`timer` provides platform-independent time measurement primitives.
+
+## Exposed API
+
+```c
+MOCKABLE_FUNCTION(, TIMER_HANDLE, timer_create_new);
```

Comment:
>TIMER_HANDLE [](start = 20, length = 12)

missing typedef?

Reply:
Added

---
In reply to: [512801686](https://github.com/Azure/azure-c-pal/pull/48#discussion_r512801686) [](ancestors = 512801686)